pkg/controller/deployment/deployment_controller_test.go

```diff
@@ -193,20 +193,9 @@ func (f *fixture) run(deploymentName string) {
 		f.t.Errorf("error syncing deployment: %v", err)
 	}
 
-	actions := f.client.Actions()
+	actions := filterInformerActions(f.client.Actions())
 	informerActions := 0
 	for i, action := range actions {
-		if len(action.GetNamespace()) == 0 &&
-			(action.Matches("list", "pods") ||
-				action.Matches("list", "replicasets") ||
-				action.Matches("list", "deployments") ||
-				action.Matches("watch", "pods") ||
-				action.Matches("watch", "replicasets") ||
-				action.Matches("watch", "deployments")) {
-			informerActions++
-			continue
-		}
-
 		if len(f.actions)+informerActions < i+1 {
```

Comment:
informerActions will be always 0 now. I think you can remove `informerActions` now. It will make it more readable.